src/webgpu/listing_meta.json

```diff
@@ -1260,7 +1260,10 @@
   "webgpu:shader,execution,expression,call,builtin,determinant:abstract_float:*": { "subcaseMS": 15.306 },
   "webgpu:shader,execution,expression,call,builtin,determinant:f16:*": { "subcaseMS": 37.192 },
   "webgpu:shader,execution,expression,call,builtin,determinant:f32:*": { "subcaseMS": 10.742 },
-  "webgpu:shader,execution,expression,call,builtin,distance:abstract_float:*": { "subcaseMS": 14.503 },
+  "webgpu:shader,execution,expression,call,builtin,distance:abstract_float:*": { "subcaseMS": 0.000 },
```

Comment:
The latest version of `run-cts` should have put in real timings here?

Reply:
This is from a chicken-egg problem with my workflow, since `npm run fix` will fail without values, which I use a lot during dev. Forgot to go back in and remove the lines/run-cts. Fixed.